packages/wrangler/package.json

```diff
@@ -138,6 +139,7 @@
 		"patch-console": "^1.0.0",
 		"pretty-bytes": "^6.0.0",
 		"prompts": "^2.4.2",
+		"react": "^17.0.0",
```

Comment:
I'd love to figure out if we can remove this. What's causing the need for it? Do we have a stray eslint config that still references react or something like that?

Reply:
We can remove this - we just get a peer-dependency warning on pnpm i, which we can ignore.